### ISSUE TITLE
Automatic unsuspend too agressive

### ISSUE THREAD

**Issue by lambdacalculator** [deanoemcke/thegreatsuspender#114]
I have "Automatically unsuspend when tab gains focus" checked in the settings, and I find its action too aggressive. For example,
- If I close a tab, then the tab that happens to get focus next is reloaded, even though I didn't purposely focus this tab. (Reloading this new tab should require an extra click.)
- If I suspend a tab, click the TGS icon to bring up the menu, and then click inside the suspended window to dismiss the TGS menu, then the tab is reloaded. (I feel that this action should just dismiss the menu without reloading the tab.)

In short, I feel that tabs should only be reloaded when the user explicitly focuses them, not as a side-effect of other operations.

**Comment by EarMaster** [comment]
Or maybe just have an option like "Automatically unsuspend when tab gains focus after:" and then allow the users to select a number of seconds to wait before a tab will be unsuspended.

**Comment by deanoemcke** [comment]
This has now been fixed. That is, the unsuspension caused by the accidental focus gained after closing a neighbouring tab.
I will not address the other issue (clicking to remove the popup) as it's more difficult to address and also, there is an alternative of NOT clicking on the suspended tab to remove the popup.

**Comment by es6Test** [comment]
How can I disable the clicking to remove the popup?

**Comment by deanoemcke** [comment]
Available in the chrome webstore in v6.30

**Comment by deanoemcke** [comment]
This functionality was reverted in v7. See this issue for more information: #519.
I have opened another new issue that suggests this functionality should be configurable: #747
